tests/unit/backend/corpora/common/entities/test_dataset_asset.py

```diff
@@ -39,7 +39,7 @@ def test__generate_file_url__OK(self):
         self.assertIn(file_name, url)
         self.assertIn("Expires=", url)
 
-    @mock.patch("backend.corpora.common.entities.dataset_asset.s3_client")
+    @mock.patch("backend.corpora.common.entities.dataset_asset.s3.client")
```

Comment:
I must really have more python to learn--where is this coming from? I see no `s3` attribute/method/class in `DatasetAsset` or `Entity` 😅

Reply:
```make local-unit-test-backend path='tests.unit.backend.corpora.common.entities.test_dataset_asset.test__generate_file_url__ERROR'```

gives 

```AttributeError: module 'tests.unit.backend.corpora.common.entities.test_dataset_asset' has no attribute 'test__generate_file_url__ERROR'```